runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkBatchTransformTranslators.java

```diff
@@ -772,14 +771,7 @@ public void translateNode(
                 doFnSchemaInformation,
                 sideInputMapping);
 
-        if (FlinkCapabilities.supportsOutputDuringClosing()) {
-          outputDataSet =
-              new FlatMapOperator<>(inputDataSet, typeInformation, doFnWrapper, fullName);
-        } else {
-          // This can be removed once we drop support for 1.8 and 1.9 versions.
-          outputDataSet =
-              new MapPartitionOperator<>(inputDataSet, typeInformation, doFnWrapper, fullName);
-        }
+        outputDataSet = new FlatMapOperator<>(inputDataSet, typeInformation, doFnWrapper, fullName);
```

Comment:
I think we can now remove MapPartition interface from https://github.com/apache/beam/blob/master/runners/flink/src/main/java/org/apache/beam/runners/flink/translation/functions/FlinkDoFnFunction.java#L65

Reply:
Oh good catch. I will fix, thx!